.rubocop.yml

```diff
@@ -0,0 +1,29 @@
+Lint/UnusedMethodArgument:
+    AllowUnusedKeywordArguments: true
```

Comment:
You don't control the code that calls the function, so it's not possible to refactor.

This situation is common, which is why the check doesn't make sense. False positives are worse than false negatives when linting.